android-core/src/androidTest/kotlin/com.mparticle/MParticleTest.kt

```diff
@@ -329,6 +331,27 @@ class MParticleTest : BaseCleanStartedEachTest() {
         Assert.assertNull(MParticle.getInstance()!!.mMessageManager.location)
     }
 
+    @Test
+    fun testEnableLocationTrackingAndDisableLocationTracking() {
+        val location = Location("")
+        val mp = MParticle.getInstance()
+        mp!!.enableLocationTracking(LocationManager.NETWORK_PROVIDER, 30*1000, 1000)
+        mp!!.setLocation(location)
+        Assert.assertEquals(location, mp!!.mMessageManager.location)
+        mp.disableLocationTracking()
+        Assert.assertNull(mp.mMessageManager.location)
+    }
+
+    @Test
+    fun testEnableLocationTracking() {
+        val location = Location("")
+        val mp = MParticle.getInstance()
+        mp!!.enableLocationTracking(LocationManager.NETWORK_PROVIDER, 30*1000, 1000)
+        mp!!.setLocation(location)
+        Assert.assertEquals(location, mp!!.mMessageManager.location)
+        Assert.assertNotNull(mp.mMessageManager.location)
+    }
```

Comment:
This looks fine.  I would swap these tests, so that `testEnableLocationTracking` comes before `testEnableLocationTrackingAndDisableLocationTracking`. Not a huge deal, but I like to go from less specific to more specific tests.